src/AddTodoForm.js

```diff
@@ -19,14 +20,12 @@ function AddTodoForm({ onAddTodo }) {
 
   return (
     <form onSubmit={handleAddTodo}>
-      <label htmlFor='todoTitle'>Title: </label>
-      <input
-        type='text'
-        id='todoTitle'
-        name='title'
-        value={todoTitle}
-        onChange={handleTitleChange}
-      />
+      <InputWithLabel
+        todoTitle={todoTitle}
+        handleTitleChange={handleTitleChange}
+      >
+        Title:
+      </InputWithLabel>
```

Comment:
Looking good! 👍 

You're only missing the `id` prop here. That will allow two instances of the `InputWithLabel` to be created. Also, you can still pass in an `isFocused` prop to so the component knows if it should autoFocus

Reply:
I'm not sure which `id` would be that, is the `todoTitle.id` ?

Reply:
Nice! That's what programming is about. Reading documentation ahahah